src/stack.ts

```diff
@@ -238,6 +238,7 @@ export class App
         }
 
         const app = new cdk.App({
+            outdir: 'cdk.out',
```

Comment:
Alternatively we could keep the default as is and instruct people that use Docker assets to provide this property.